src/cli/helpers.ts

```diff
@@ -216,6 +217,34 @@ function emitTestCaseHooks(
     })
 }
 
+function emitTestStepHooks(
+  supportCodeLibrary: ISupportCodeLibrary,
+  eventBroadcaster: EventEmitter
+): void {
+  ;[]
+    .concat(
+      supportCodeLibrary.beforeTestStepHookDefinitions,
+      supportCodeLibrary.afterTestStepHookDefinitions
+    )
+    .forEach((testStepHookDefinition: TestStepHookDefinition) => {
+      eventBroadcaster.emit(
+        'envelope',
+        messages.Envelope.fromObject({
+          hook: {
```

Comment:
I don't think step hooks should be emitted as cucumber-messages doesn't have a way to differentiate these from scenario hooks. Also we don't independently report a result for step hooks with them being bundled into steps. 

I think initially these should be more internal to cucumber-js. After some usage we can see if we want to keep it that way or find a way to report them (and their results) in messages.